pages/contact.html

```diff
@@ -57,5 +57,47 @@ <h1 class="ui dividing header" style="padding-top:8px">School Location</h1>
     <p>School Location on Google maps</p>
     <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9769.151915664062!2d-7.1166274!3d52.2563154!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf8355ef4efe9f1a4!2sMount+Sion+CBS+Secondary+School!5e0!3m2!1sen!2sie!4v1523822538739" width="100%" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
 </div>
+
+<div class="ui inverted vertical footer segment">
```

Comment:
I think the inverted footer goes against the style of the wireframe, simply remove inverted for now

Reply:
If its not inverted the words wont appear on the page cause of the font colour being white.

Reply:
@maciejmarchel12 `inverted` sets the background to black and font to white, by removing `inverted` it sets the background to white and the font to black